settings/DevHome.Settings/Views/AccountsPage.xaml.cs

```diff
@@ -84,7 +103,24 @@ private async void AddDeveloperId_Click(object sender, RoutedEventArgs e)
         {
             if (addAccountButton.Tag is AccountsProviderViewModel accountProvider)
             {
-                await ShowLoginUIAsync("Settings", this, accountProvider);
+                if (accountProvider.DeveloperIdProvider.GetAuthenticationExperienceKind() == Microsoft.Windows.DevHome.SDK.AuthenticationExperienceKind.CardSession)
+                {
+                    await ShowLoginUIAsync("Settings", this, accountProvider);
+                }
+                else if (accountProvider.DeveloperIdProvider.GetAuthenticationExperienceKind() == Microsoft.Windows.DevHome.SDK.AuthenticationExperienceKind.CustomProvider)
+                {
+                    IntPtr windowHandle = IntPtr.Zero;

```

Comment:
Promote to a method to avoid duplicate code

Reply:
Addressed in recent commit. Thanks!